package.json

```diff
@@ -37,7 +37,7 @@
     "@types/history": "^4.7.8",
     "@types/jest": "^26.0.20",
     "@types/node": "^14.14.28",
-    "@types/react": "^17.0.2",
+    "@types/react": "^17.0.1",
```

Comment:
trøbbel ?

Reply:
Jepp, trodde det var den som ødela for aws. Er oppgradert igjen.